external-crates/move/crates/move-compiler/src/expansion/ast.rs

```diff
@@ -597,21 +536,11 @@ impl Hash for Address {
 // impls
 //**************************************************************************************************
 
-impl Attribute_ {
-    pub fn attribute_name(&self) -> &Name {
-        match self {
-            Attribute_::Name(nm)
-            | Attribute_::Assigned(nm, _)
-            | Attribute_::Parameterized(nm, _) => nm,
-        }
-    }
-}
-
 impl Attributes {
     pub fn is_test_or_test_only(&self) -> bool {
```

Comment:
A note for later, maybe this should be renamed `is_test_mode_enabled` or something more "mode" centric